bodo/hiframes/pd_dataframe_ext.py

```diff
@@ -3809,6 +3816,7 @@ def to_parquet_overload(
             + str(get_overload_const_str(compression))
         )
 
+    part_col_idxs = None
```

Comment:
I'm just curious, why was this necessary? Linter complaints?

Reply:
This was a hold-over from when I was creating the hash from the vars used to create the function and so it had to exist.  Removed it since using different approach now.